CHANGELOG.md

```diff
@@ -76,6 +76,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Fixed: Starting in Spazer Beam Chamber now places Samus in the correct room.
 
+#### Area 4 Crystal Mines
+
+- Fixed: A typo in the room name "Gamma+ Arena" has been changed from "Gamma Arena".
+- Fixed: The Gamma+ Metroid being classified as a Gamma Metroid, thus having the wrong requirements.
```

Comment:
We have a setting that changes the restricted placement to be on seperate types of metroids
This needs to be changed there as well, no?